README.md

```
@@ -59,48 +61,147 @@ resource "aws_instance" "example" {
+The DNS name example demonstrates how to use Terragrunt with this module. In the example:
+
+- A Terragrunt configuration (in examples/dns_name/terragrunt.hcl) sets up local variables such as account ID, tenant, region, and domain details.
+- The inputs block passes these values to the module, including naming conventions, regex filters, and tag configurations.
+- The module is used in multiple files:
+  - \_context.tf initializes the standard context required by the module.
+  - dns.tf shows a simple module call that generates a DNS name based on your provided configuration.
+  - \_outputs.tf outputs key properties such as the generated ID, DNS name, and format.
+
+This setup allows you to test-deploy resources in a sandbox environment and clearly see how the module processes input variables to generate a consistent naming and tagging policy.
+
+# Roadmap
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)